backend/src/main/java/com/google/blackswan/servlets/CronServlet.java

```diff
@@ -64,7 +68,7 @@ private void clearCurrentAlertsInDatastore() {
 
   /** Store alerts from simpleGenerator into the datastore. */
   private void storeAlertsInDatastoreSimple() {
-    AlertGenerator simpleGenerator = new SimpleAlertGenerator(SimpleAnomalyGenerator.createGenerator());
+    AlertGenerator simpleGenerator = new AdvanceAlertGenerator(ANOMALY_TYPES);
```

Comment:
in the "future", ANOMALY_TYPES could have more datainfos, which could also come from user input?

Reply:
yes, you can add more `DataInfo`s and it would work right now. We can have it specified by the user or it can come from a config file. (I am working on the config file part that can be shared between `java` and `python` service.)